token-metadata/program/src/state/programmable.rs

```diff
@@ -27,28 +28,45 @@ pub const TOKEN_RECORD_SEED: &str = "token_record";
 
 pub const TOKEN_STATE_INDEX: usize = 2;
 
-pub const TOKEN_RECORD_SIZE: usize = 1 // Key
-+ 1  // bump
-+ 1  // state
-+ 9  // rule set revision
-+ 33 // delegate
-+ 2; // delegate role
+pub const LOCKED_TRANSFER_SIZE: usize = 33; // Optional Pubkey
 
+pub const TOKEN_RECORD_SIZE: usize = 1 // Key
++ 1   // bump
++ 1   // state
++ 9   // rule set revision
++ 33  // delegate
++ 2   // delegate role
++ 33; // locked transfer
```

Comment:
We might want to allocate extra padding to prevent the need for additional migration functions in the future.

Reply:
In this case padding is not going to avoid having the manual deserialization since there are already accounts that are shorter and might not get updated in the near future.